### PR TITLE
Set log level using env variable

### PR DESCRIPTION
**Pull request by F3real**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -53,7 +55,19 @@ fn main() {
         output: vec![fern::OutputConfig::stdout(), fern::OutputConfig::file("sccache2.log")],
         level: log::LogLevelFilter::Trace,
     };
-    if let Err(e) = fern::init_global_logger(logger_config, log::LogLevelFilter::Warn)
+
+    // Set current log level depending on environment variable
+    let log_level = match &*env::var("SCCACHE2_LOG_LEVEL").unwrap_or("warn".to_string()).to_lowercase() {
```
Comment (luser) [inline src/main.rs]: Could you move this all into a separate function, like `fn init_logging()`, and just call that from main? I like having main with a very simple body.

**Comment by F3real** [comment]
@luser I've moved code to new function

**Comment by luser** [comment]
Thanks! I have an idea for a follow-up to make logging even more useful. I will file a new issue.

**Comment by armenzg** [comment]
\o/

**Pull request merged**